tools/stress-cluster/cluster/README.md

```diff
@@ -1,160 +1,160 @@
-This directory contains [Azure Bicep](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview)
+Table of Contents
+* [Layout](#layout)
+* [Dependencies](#dependencies)
+* [Deploying Cluster(s)](#deploying-clusters)
+   * [Dev Cluster](#dev-cluster)
+   * [Test Cluster](#test-cluster)
+   * [Prod Cluster](#prod-cluster)
+   * [Local Cluster](#local-cluster)
+* [Development](#development)
+   * [Bicep templates](#bicep-templates)
+   * [Helm templates](#helm-templates)
+
+
+# Layout
+
+This directory contains all configuration used for stress test cluster buildout (azure and kubernetes buildout), as well
+as a set of common stress test config boilerplate (helm library).
+
+The `./azure` directory contains [Azure Bicep](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview)
 files for deploying Azure resources (mainly [AKS clusters](https://azure.microsoft.com/en-us/services/kubernetes-service/)
 to support stress testing (for dev/test and/or production).
 
 Azure Bicep comes pre-installed with the Azure CLI, and is a DSL for generating ARM templates.
 
+The `./kubernetes/stress-infrastructure` directory contains a helm chart for deploying the core services
+that must be installed into any stress cluster: chaos-mesh (for chaos) and stress-watcher (for event handling like chaos
+resource start and resource group cleanup).
+
+The `./kubernetes/stress-test-addons` directory contains a [library chart](https://helm.sh/docs/topics/library_charts/)
+for use by stress test packages. This common set of config boilerplate simplifies stress test authoring, and makes it
+easier to make and roll out config changes to tests across repos by using helm chart dependency versioning.
+
+
 # Dependencies
 
+- [Powershell Core](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-linux?view=powershell-7.1#ubuntu-2004) (if using Linux)
```

Comment:
I'd recommend powershell core for all OS's, as there is a lot of improvements in networking over top of windows powershell as well.